src/test/java/org/apache/commons/imaging/formats/ico/IcoImageParserTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.ico;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.apache.commons.imaging.ImagingException;
+import org.apache.commons.imaging.test.TestResources;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+public class IcoImageParserTest {
+
+    /**
+     * For https://issues.apache.org/jira/browse/IMAGING-373.
+     * <p> There is a problem with loading bitmap stored in given ICO file, so the exception is originally thrown by BmpImageParser.
```

Comment:
Close HTML tags. No need for a space after `<p>`.

Reply:
Fixed.

I haven't noticed any plugin like https://github.com/diffplug/spotless in the project, I might just missed it. Apparently I'm relying too much on such tools :)